index.html

```diff
@@ -99,50 +111,60 @@ <h3 class="projecthead">Professional Art Printing Data</h3>
-                    <li class="skillitem">Terminal</li>
-                </ul>
+                <div class="language">
+                    <span class="diamond"></span>
+                    <h3 class="aboutmesub">Language</h3>
+                    <ul class="skilllist">
+                        <li class="skillitem">Javascript</li>
+                        <li class="skillitem">Ruby</li>
+                        <li class="skillitem">Html</li>
+                        <li class="skillitem">Css</li>
+                    </ul>
+                </div>
+                <div class="framework">
+                    <span class="square"></span>
+                    <h3 class="aboutmesub">Frameworks</h3>
+                    <ul class="skilllist">
+                        <li class="skillitem">Bootstrap</li>
+                        <li class="skillitem">Ruby on Rails</li>
+                        <li class="skillitem">RSPec</li>
+                        <li class="skillitem">CapyBara</li>
+                        <li class="skillitem">Selenium</li>
+                    </ul>
+                </div>
+                <div class="git">
+                    <span class="circle"></span>
+                    <h3 class="aboutmesub">Skills</h3>
+                    <ul class="skilllist">
+                        <li class="skillitem">Codekit</li>
+                        <li class="skillitem">Github</li>
+                        <li class="skillitem">Codepen</li>
+                        <li class="skillitem">Gitlab</li>
+                        <li class="skillitem">Terminal</li>
+                    </ul>
+                </div>
             </div>
         </section>
         <section class="forms">
-            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in </h2>
+            <h2 class="formhead">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
             <form action="https://formspree.io/f/mgedjavk" method="post">
-                <input type="text" name="name" id="name" placeholder="Full name" required maxlength="30">
+                <input class="nodisplay" type="text" name="name" id="name" placeholder="Full name" required maxlength="30">
+                <input class="nodisplaym" type="text" name="firstname" id="firstname" placeholder="First name" required maxlength="30">
+                <input class="nodisplaym" type="text" name="lastname" id="lastname" placeholder="Last name" required maxlength="30">
                 <input type="email" name="email" id="email" placeholder="Email address" required>
                 <textarea name="msg" id="msg" placeholder="Enter text here" required maxlength="500"></textarea>
-                <button type="submit">Get in touch</button>
+                <button class="formbutton" type="submit">Get in touch</button>
             </form>
         </section>
         <footer>
```

Comment:
Please make sure your icons' colors match that of the template design in both the mobile and desktop version.